docs/resources/resources-glossary.md

```diff
@@ -0,0 +1,132 @@
+
+## host
+
+In the Office Add-ins documentation, **host** typically refers to an Office application. The Office applications, or hosts, that support Office Add-ins are: Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [application](#application), [client](#client), [Office application, Office client](#office-application-office-client).
+
+## Office application, Office client
+
+In the Office Add-ins documentation, **Office client** refers to an Office application. The Office applications, or clients, that support Office Add-ins are: Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [application](#application), [client](#client), [host](#host).
+
+## platform
+
+In the Office Add-ins context, a **platform** usually refers to the operating system running an add-in. Platforms that support Office Add-ins are: Windows, Mac, iPad, and web browsers.
+
+## requirement set
+
+[!include[Requirement set note](../includes/office-js-requirement-sets.md)]
+
+## ribbon, ribbon button
+
+A **ribbon** is a command bar that organizes an application's features into a series of tabs or buttons at the top of a window. A **ribbon button** is one of the buttons within this series.
+
+## runtime
+
+In the Office Add-ins context, a **runtime** is a lifecycle, or the time during which an application is running.
+
+## task pane
+
+Task panes are interface surfaces that typically appear on the right side of the window within Word, PowerPoint, Excel, and Outlook. Task panes give users access to interface controls that run code to modify documents or emails, or display data from a data source. Use task panes when you don't need to embed functionality directly into the document. See [Task panes in Office Add-ins](../design/task-pane-add-ins.md) to learn more.
+
+## tutorial
+
+A **tutorial** is a teaching aid designed to help people learn to use a product or procedure. In the Office Add-ins context, a tutorial guides an add-in developer through the complete add-in development process for a particular application, such as Excel. This involves following 20 or more steps and is a greater time investment than a [quickstart](#quickstart).
+
+See also: [quickstart](#quickstart).
+
+## quickstart
+
+A **quickstart** is a a high level description of key skills and knowledge required for the basic operation of a particular program. In the Office Add-ins documentation, a quickstart is an introduction to developing an add-in for a particular application, such as Outlook. A quickstart contains series of steps that an add-in developer can complete in approximately 5 minutes, resulting in a functioning add-in.
+
+See also: [tutorial](#tutorial).
+
+## UI-less custom function
+
+A **UI-less custom function** is a custom functions add-in that doesn't have a task pane or other user-interface elements.
```

Comment:
I see your point, but since custom functions is never hyphenated I want to avoid it. Do you think italicizing "custom functions add-in" would help in this context?

Reply:
I suggested the hyphen based on how I understood what's said in the [MS style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28765) re: noun modifiers